packages/web/src/js/Tooltip.ts

```diff
@@ -36,6 +81,22 @@ class Tooltip extends BaseComponent {
     }
   }
 
+  isPlacementControlled() {
+    const { placementControlled } = this.config as Config;
+
+    /* eslint-disable no-undefined */
+    return placementControlled !== undefined;
```

Comment:
To use this without the `eslint-disable` you can `typeof placementControlled !== "undefined"` or `placementControlled != null`, I think.

Reply:
`undefined` changed to `false`